package.json

```diff
@@ -179,6 +179,7 @@
     "unit:scripts": "time c8 -o ./coverage/scripts-unit tap --test-regex='(\\/|^bin\\/test\\/.*\\.test\\.js)$' --timeout=180 --no-coverage --reporter classic",
     "update-cross-agent-tests": "./bin/update-cats.sh",
     "versioned-tests": "./bin/run-versioned-tests.sh",
+    "versioned-tests-selected": "./bin/run-versioned-tests.sh langchain",
```

Comment:
Please remove this line. Use `npm run versioned:internal:major langchain` instead.